src/test/tests/maven.test.ts

```diff
@@ -360,4 +353,61 @@ describe('Maven Tests', async () => {
             ]
         ];
     }
+
+    describe('Update to fixed version', async () => {
```

Comment:
We already setup a test environment for 3 projects that are not relevant to the added tests for the dependency update.
Can we add a new test file "mavenUpdate.test.ts" and move all of the new tests there?